src/core/XRPValue.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const GlobalBigNumber = require('bignumber.js');
+const BigNumber = GlobalBigNumber.another({
+  ROUNDING_MODE: GlobalBigNumber.ROUND_HALF_UP,
+  DECIMAL_PLACES: 40
+});
+
+const Value = require('./value').Value;
+
+class XRPValue extends Value {
+
+  constructor(value) {
```

Comment:
It might be a good idea to throw an exception in the constructor if `value` has more than 6 digits of precision past the decimal point. This would help catch bugs where an IOUValue is being cast to an XRPValue.